tests/test_dependencies_events.py

```diff
@@ -50,3 +55,49 @@ async def test_event(self):
         self.assertEqual(event["requestContext"]["path"], "/ping/pong")
         self.assertEqual(event["requestContext"]["httpMethod"], "GET")
         self.assertEqual(event["requestContext"]["protocol"], "HTTP/1.1")
+
+
+class TestSQS(unittest.IsolatedAsyncioTestCase):
+    @patch("uuid.uuid4", return_value=uuid.UUID("12345678123456781234567812345678"))
+    async def test_event(self, mock_uuid):
+        data = {
+            "message": {
+                "data": "aGVsbG8=",
+                "attributes": {"foo": "bar"},
+                "messageId": "10519041647717348",
+                "publishTime": "2024-02-22T15:45:31.346Z",
```

Comment:
Esse timestamp está sendo referenciado em mais de um lugar do teste, acho melhor armazenar em uma variável e utilizar a variável onde o timestamp é necessário. Desta forma se for necessário ajustar o timestamp basta ajustar a variável.